src/openapi/streaming/streaming.ts

```diff
@@ -1283,7 +1283,7 @@ class Streaming extends MicroEmitter<EmittedEvents> {
             // disconnecting *should* shut down all subscriptions. We also delete all below.
             // So mark the subscription as not having a connection and reset it so its state becomes unsubscribed
             subscription.onConnectionUnavailable();

```

Comment:
Do we still need to call onConnectionUnavailable if we dispose afterwards? Or is it kept to ensure we have logged info about specific disposal scenario?